impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/instance/BuiltinInstanceDecoratorTest.java

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-package org.jboss.cdi.tck.tests.decorators.builtin.instance;
+package org.jboss.cdi.tck.tests.full.decorators.instance;
```

Comment:
`decorators.builtin` should IMHO move to `full.decorators.builtin`, not `full.decorators`.

Reply:
Done